com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1296,6 +1296,7 @@ private void OnNetworkPostLateUpdate()
             {
                 MessagingSystem.ProcessSendQueues();
                 NetworkMetrics.UpdateNetworkObjectsCount(SpawnManager.SpawnedObjects.Count);
+                NetworkMetrics.UpdateConnectionsCount((IsServer) ? ConnectedClients.Count : 1);
```

Comment:
A bit meta, but would you mind adding a Jira ticket for the work to potentially get the connected client count for clients as well?

Reply:
Good idea!